server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraConfTest.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.james;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+import java.nio.charset.Charset;
+
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.james.modules.mailbox.CassandraSessionConfiguration;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.google.common.base.Joiner;
+
+public class CassandraConfTest {
+
+    private static final int IMAP_PORT = 1143;
+    private static final int CASSANDRA_PORT = 9042;
+	private GuiceJamesServerImpl jamesServer;
+	private SocketChannel socketChannel;
+
+
+    private PropertiesConfiguration getCassandraConfigurationForDocker(String... ips) {
+        PropertiesConfiguration configuration = new PropertiesConfiguration();
+        
+        configuration.addProperty("cassandra.ip", Joiner.on(',').join(ips));
+        configuration.addProperty("cassandra.port", CASSANDRA_PORT);
+        configuration.addProperty("cassandra.keyspace", "apache_james");
+        configuration.addProperty("cassandra.replication.factor", 1);
+        configuration.addProperty("cassandra.retryConnection.maxRetries", 10);
+        configuration.addProperty("cassandra.retryConnection", 5000);
+
+        return configuration;
+    }
+    private final DockerCassandraRule dockerCassandraRule = new DockerCassandraRule();
+	@Rule public CassandraJmapTestRule cassandraJmapTestRule = new CassandraJmapTestRule(dockerCassandraRule, new EmbeddedElasticSearchRule());
```

Comment:
Line break between @Rule and public ? Line break between private final and public variables ?